cves/CVE-2012-0022.yml

```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 189
```

Comment:
I believe CWE-407 is a better fit here. CWE-189 applies to "improper calculations." Hash table collisions are better described by "inefficient worst-case computational conplexity."